website/docs/r/service.html.markdown

```diff
@@ -10,6 +10,14 @@ description: |-
 
 This resource allows you to manage a service account - an account for machines.
 
+## Example Usage
+
+```terraform
+resource "secrethub_service" "demo_service_account" {
+  repo        = "username/demo"
```

Comment:
Maybe use `workspace` instead of `username`, as this can be used in shared workspaces as well. That's the most common use-case.